migration/src/bin/migrate_to.rs

```diff
@@ -0,0 +1,393 @@
+use clap::{builder::PossibleValuesParser, command, Parser, ValueEnum};
+use migration::{
+    sea_orm::{ConnectOptions, Database, DatabaseConnection, EntityTrait, QueryOrder},
+    MigratorTrait,
+};
+use sea_orm_migration::{prelude::*, seaql_migrations};
+use tracing::{info, Level};
+
+#[derive(Copy, Clone, ValueEnum, Debug)]
+enum Direction {
+    Up,
+    Down,
+}
+
+/// Wraps the SeaORM migration library to provide additional features. Lets
+/// you bring migrations up or down to a particular version. Allows dry-run
+/// of migrations.
+#[derive(Parser, Debug)]
+#[command(about, version)]
+struct Args {
+    #[arg(value_enum)]
+    direction: Direction,
+    #[arg(value_parser = available_migrations())]
+    target_version: String,
+    #[arg(short, long)]
+    dry_run: bool,
+    #[arg(short = 'u', long, env = "DATABASE_URL")]
+    database_url: String,
+}
+
+#[async_std::main]
+async fn main() -> Result<(), Error> {
+    let args = Args::parse();
+    tracing_subscriber::fmt()
+        .with_max_level(Level::DEBUG)
+        .init();
+
+    let db = Database::connect(ConnectOptions::new(args.database_url)).await?;
+    migrate_to::<migration::Migrator>(&db, args.dry_run, args.direction, &args.target_version)
+        .await?;
+    Ok(())
+}
+
+async fn migrate_to<M: MigratorTrait>(
+    db: &DatabaseConnection,
+    dry_run: bool,
+    dir: Direction,
```

Comment:
```suggestion
    direction: Direction,
```

Just because if I see `dir`, I think "directory".